magda-migrator-content-db/sql/V1_8__footer_default_config.sql

```diff
@@ -0,0 +1,43 @@
+INSERT INTO "public"."content" ("id", "type", "content") VALUES ('footer/navigation/categories/medium/data.gov.au', 'application/json', '{
+  "label": "Data.gov.au",
```

Comment:
Internal magda shouldn't reference data.gov.au

Reply:
Good picks 👍 Has updated this